spec/storage/file_spec.rb

```diff
@@ -6,49 +6,63 @@
   let(:file){ CarrierWave::Storage::PostgresqlLo::File.new(uploader) }
   let(:tempfile){ stub_tempfile('test.jpg', 'application/xml') }
   let(:file_content){ 'this is stuff' }
-  let(:identifier){ Test.connection.raw_connection.lo_creat }
+  let(:identifier) {
+    if defined?(JRUBY_VERSION)
+      Test.connection.raw_connection.connection.getLargeObjectAPI.createLO
+    else
+      Test.connection.raw_connection.lo_creat 
+    end
+  }
 
   describe "#delete" do
     it("should delete the file using the lo interface") do
-      file.write(tempfile)
-      file.delete
+      ActiveRecord::Base.transaction {
```

Comment:
AR-JDBC will fail these tests in Auto-Commit mode. Using a transaction here is the easiest way to disable auto-commit mode.